src/components/Content.js

```diff
@@ -1,37 +1,67 @@
-import React from "react";
+import React, { useEffect, useState } from "react";
 
 const Content = ({issues}) => {
 
+    const [isShown, setIsShown] = useState(false);
+    const [author, setAuthor] = useState([])
+
     let count=0,count2 = 0, active, closed;
     active = issues.filter(item => item.state === 'open');
     closed = issues.filter(item => item.state === 'closed');
     count = active.length;
     count2 = closed.length;
+    
+    const loadAuthor = (term) => {
+        console.log(term)
+    }
+
+    useEffect(() => {
+
+    }, [])
```

Comment:
What are these for?

Reply:
wala pa na tapos sir. wala nako na stash.